src/components/LocationCard/LocationCard.js

```diff
@@ -13,6 +13,7 @@ export const LocationCard = (props) => {
         As,
         locationId,
         locationTitle,
+        hasRadiusSlider,
```

Comment:
![Screenshot 2020-05-18 at 13 20 13](https://user-images.githubusercontent.com/19436304/82207586-5d2a5700-990a-11ea-9808-bb1327e46926.png)
The selected item is inside a gray block according to the task REACH-82. I don't know that issue or not. Could @carlobernardini please confirm.

Reply:
It's fine to keep it as is for now, let's look into this later.

Reply:
@carlobernardini should we add that comment to ONEUI-184?